modules/howtos/pages/managing-connections.adoc

```diff
@@ -161,7 +181,7 @@ It is important to make sure you are transferring the certificate in an encrypte
 If you are running on `localhost` and just want to enable TLS for a development machine, just copying and pasting it suffices
 -- _so long as you use `127.0.0.1` rather than `localhost` in the connection string_.
 This is because the certificate will not match the name _localhost_.
-Setting `TLSSkipVerify` is a workaround if you need to use ` couchbases://localhost`.
+Setting `TLSSkipVerify` is a workaround if you need to use `couchbases://localhost`.
```

Comment:
I can't find any reference to `TLSSkipVerify` in the Java SDK. I suspect the corresponding client might be `security.enableHostnameVerification` which you can set to `false`, either in the connection string or configuring the cluster environment's SecurityConfig`.

I was going to cite a documentation reference, but it looks like this setting is undocumented? Here's where I looked for it: https://docs.couchbase.com/java-sdk/current/ref/client-settings.html#security-options

Or... if the intent is to completely disable all TLS verification and trust any certificate, the way to do that in Java today is to configure the environment'd SecurityConfig to use `InsecureTrustManagerFactory.INSTANCE`